qdev_wrappers/fitting/AnalysisToSQL.py

```diff
@@ -0,0 +1,151 @@
+
+
+import sqlite3
+import dill
+from qdev_wrappers.fitting.Fitclasses import T1, T2
+
+
+def is_table(tablename, cursor):  # Checks if table already exists. Assumes database connection and cursor already established
+
+    table_count = "SELECT count(*) FROM sqlite_master WHERE type = 'table' AND name='{}'".format(tablename)
+    execute = cursor.execute(table_count)
+    count = execute.fetchone()[0]
+
+    if count == 0:
+        return False
+
+    if count == 1:
+        return True
+
+
+def make_table(tablename, cursor):
+    name = tablename
+    n = 0
+
+    while is_table(name, cursor):
+        n += 1
+        name = "{}_{}".format(tablename, n)
+    else:
+        cursor.execute('CREATE TABLE {} (id INTEGER)'.format(name))
+
+    return name
+
+
+def fit_to_SQL(fit):    #it would be an improvement if it were able to get the fitclass from the fit information
```

Comment:
I would <3 <3 <3 some more comments in this function, I know that in theory you shouldn't need comments to explain what you are doing but a few here to break up this rather large function and explain the different sections would be great, especially because as it stands I don't know what the slightly monstrous 'fit' contains/needs to contain.  Also a comment at the start of each function to specify the arguments and broadly what is done/returned would be fab.

Reply:
The 'fit' is the fit dictionary, as returned by the Fitter. The comment at the very beginning is misleading because its a relic of a previous version of the function that had multiple arguments. I've deleted it, reorganized the function a bit and added more relevant comments, so I think it should be more readable now.